lib/rules/cli/4.3.yaml

```diff
@@ -0,0 +1,1353 @@
+---
```

Comment:
@xiaojiey  We already have 4.x rules.
```
[lxia@ocp verification-tests]$ ls lib/rules/cli/4.*
lib/rules/cli/4.1.yaml  lib/rules/cli/4.2.yaml  lib/rules/cli/4.3.yaml  lib/rules/cli/4.4.yaml
```

Any commands/options that you need does exist ? Add it into 4.1.yaml

Reply:
@xiaojiey yes, agree with @liangxia

Reply:
ls -l ls lib/rules/cli/4.*, you will see symlinks